### PR TITLE
Support cassandra-stress new version (3.0) parameters

### PR DESCRIPTION
**Pull request by fruch**

### CODE REVIEW

File: ccmlib/node.py [hunk 1]
```diff
@@ -1083,6 +1083,24 @@ def stress(self, stress_options=None, capture_output=False, **kwargs):
...
+
+        # handle cassandra-stress >= 3.11 parameters
+        has_limit = any(['limit=' in o for o in stress_options])
+        has_throttle = any(['throttle=' in o for o in stress_options])
+        if has_limit or has_throttle:
+            args = [stress, 'version']
+            p = subprocess.Popen(args, cwd=common.parse_path(stress),
+                                 stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+                                 **kwargs)
+            stdout, stderr = p.communicate()
```
Comment (bentsi) [inline ccmlib/node.py]: if you don't use `stdout` better to replace by `_`

**Comment by juliayakovlev** [comment]
Can you add some explanation about this, please?

**Comment by bentsi** [comment]
@juliayakovlev in cassandra-stress >3.11 `limit` argument was renamed to `throttle`, so we need handle this situations since most of the tests are written using oldest version of c-s.

File: ccmlib/node.py [hunk 2]
```diff
@@ -1083,6 +1083,24 @@ def stress(self, stress_options=None, capture_output=False, **kwargs):
...
+
+            if p.returncode == 1 and has_throttle:
```
Comment (juliayakovlev) [inline ccmlib/node.py]: Why that? Does the c-s > 3.11 have no "version" argument?

Comment (bentsi) [reply ccmlib/node.py]: exactly opposite, c-s> 3.11 has *version* argument and  c-s<3.11 doesn't have *version* argument

**Pull request merged**